src/Umbraco.Cms.StaticAssets/Umbraco.Cms.StaticAssets.csproj

```diff
@@ -1,20 +1,11 @@
 <Project Sdk="Microsoft.NET.Sdk.Razor">
-
   <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
+    <Title>Umbraco CMS - Static assets</Title>
+    <Description>Contains the static assets needed to run Umbraco CMS.</Description>
     <AddRazorSupportForMvc>true</AddRazorSupportForMvc>
-    <PackageId>Umbraco.Cms.StaticAssets</PackageId>
-    <Description>Contains the static assets that is required to run Umbraco CMS.</Description>
     <StaticWebAssetBasePath>/</StaticWebAssetBasePath>
   </PropertyGroup>
 
-  <ItemGroup>
-    <None Include="buildTransitive\**\*.*">
-      <Pack>true</Pack>
-      <PackagePath>buildTransitive</PackagePath>
-    </None>
-  </ItemGroup>
```

Comment:
Just for my own edification since I'm not that well versed in all of this csproj stuff, but why are we removing this, and why was it there in the first place? 😄

Reply:
Ahh right of course I see, that's my bad 😄